globus_sdk/auth/token_response.py

```diff
@@ -30,6 +33,7 @@ def __init__(self, *args, **kwargs):
         GlobusHTTPResponse.__init__(self, *args, **kwargs)
         # dict used to avoid re-computing self.by_resource_server
         self._by_resource_server = None
+        self._by_resource_server = self.by_resource_server
```

Comment:
I think we also need to do this in the Dependent Token response class, to avoid a delay between fetching it and computing the `expires_at_seconds` times.